src/core/qgismobileapp.h

```diff
@@ -233,6 +234,8 @@ class QFIELD_CORE_EXPORT QgisMobileapp : public QQmlApplicationEngine
     std::unique_ptr<DrawingTemplateModel> mDrawingTemplateModel;
     std::unique_ptr<MessageLogModel> mMessageLogModel;
 
+    std::unique_ptr<PluginManager> mPluginManager;
```

Comment:
Isn't QML taking ownership of the plugin manager?
In this case having a `unique_ptr` that also claims ownership is at risk for double deletion?

Reply:
@m-kuhn , no, the ownership stays with QgisMobileApp. The unique_ptr might be redundant here but that's what we do for the message log model object for e.g.

Reply:
You've been warned that using `std::unique_ptr` with `QObject` is dangerous :warning: if it's parented to something and something is deleted earlier, we will have a crash.

Reply:
@m-kuhn , should we update all of the relevant members then? message log model, bookmark log model, etc.?

Reply:
Or we just use simple *pointer since the objects are parented to QgisMobileApp and would live until QgisMobileApp itself is destroyed.

Reply:
Yeah, either one or the other

Reply:
@m-kuhn , OK, I'll adapt, good catch.